tests/utils.py

```diff
@@ -217,11 +220,21 @@ def create_msui_settings_file(content):
 
 
 def is_url_response_ok(url):
+    time.sleep(0.5)  # it seems we need a delay?
```

Comment:
a better idea is welcome

Reply:
Don't add a delay there. It only slows things down, the caller is responsible for retrying and waiting an appropriate amount inbetween. And I think the only call site of this function already does the wait.

Reply:
with that line, I have not any failure since > 10 runs on my MBP, without that line I get the Errors, written above. 

I will check again. I don't understand why that is needed.

Reply:
without that we can have 96 passed, 12 skipped, 38 warnings, 473 errors in 191.90s (0:03:11)
with the error Failed: Server: "mslib.mscolab.app" did not become ready in time

Reply:
at the moment we understand why this makes a difference it is solved